Build failure looks related

```
corehq.apps.export.tests.test_export_column:StockExportColumnTest.test_get_headers ... ERROR
======================================================================
ERROR: corehq.apps.export.tests.test_export_column:StockExportColumnTest.test_get_headers
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/vendor/lib/python2.7/site-packages/mock.py", line 1216, in patched
    arg = patching.__enter__()
  File "/vendor/lib/python2.7/site-packages/mock.py", line 1273, in __enter__
    self.target = self.getter()
  File "/vendor/lib/python2.7/site-packages/mock.py", line 1393, in <lambda>
    getter = lambda: _importer(target)
  File "/vendor/lib/python2.7/site-packages/mock.py", line 1126, in _importer
    thing = _dot_lookup(thing, comp, import_path)
  File "/vendor/lib/python2.7/site-packages/mock.py", line 1115, in _dot_lookup
    __import__(import_path)
ImportError: No module named Product
----------------------------------------------------------------------
Ran 303 tests in 18.908s
```

--- a/corehq/apps/export/tests/test_export_column.py
+++ b/corehq/apps/export/tests/test_export_column.py
@@ -179,8 +179,7 @@ class StockExportColumnTest(SimpleTestCase):
     domain = 'stock-domain'
 
     @patch('corehq.apps.export.models.new.StockExportColumn._get_product_name', return_value='water')
-    @patch('corehq.apps.export.models.new.Product.by_domain', return_value=[])
-    def test_get_headers(self, _, __):
+    def test_get_headers(self, _):
         column = StockExportColumn(
             domain=self.domain,
             label="Stock",

